docs/source/configuration_file.md

```diff
@@ -127,6 +127,13 @@ Uncertainty computation is performed if `n_it>0` and at least `epistemic` or `al
 - `aleatoric`: Bool. Image-based uncertainty with [test-time augmentation](https://doi.org/10.1016/j.neucom.2019.01.103).
 - `n_it`: Integer. Number of Monte Carlo iterations. Set to 0 for no uncertainty computation.
 
+## Cascaded Architecture Features
+
+### object_detection_params (Optional)
+- `object_detection_path`: String. Path to object detection model. The model's prediction will be used to generate bounding boxes.
+- `safety_factor`: List. List of length 3 containing the factors to multiply each dimension of the bounding box.
```

Comment:
Not sure: does it mean: it takes the coordinates of the prediction, compute the bounding box, then expand this bounding box with this safety factor? :-)

Reply:
Yes that's pretty much it: If the original bounding box has coordinates of 10x20x30, you can apply a safety factor of for instance 1.5, so you'll get a final bounding box size of 15x30x45.

Reply:
I added some precision in the doc :)